interactions/client.py

```diff
@@ -148,18 +150,17 @@ def decorator(coro: Coroutine) -> Any:
                     self.me = User(**data)
 
                 request = self.http.create_application_command(
-                    self.me._json["id"], data=_payload, guild_id=guild
+                    self.me.id, data=_payload, guild_id=guild
                 )
 
                 _att = self.loop.run_until_complete(request)
```

Comment:
Not against the existence of this variable but what is `_att` representing as the name? I might smooth brain 8 months later revisiting this code and wonder what the hell that means even though I understand the event loop request following it.

Reply:
`_att` (now `req_data` as of 04237d0) is the object generated from the request, if it fails, it has the code key that the lib parses.